R/intervals_support_funs.R

```diff
@@ -0,0 +1,244 @@
+# Load necessary library
+library(dplyr)
+
+#' Remove specified imputation methods from the intervals in a PKNCAdata object.
+#'
+#' @param data A PKNCAdata object containing the intervals and data components.
```

Comment:
Please ensure that it can work either on a data.frame or on a PKNCAdata object. (Other comments will relate to this, too.)